### PR TITLE
Create initial workflow

### PR DESCRIPTION
**Pull request by hbiede**
## 📑 Description

Allow population of docs from other repos

## ✅ Checks

- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

File: .github/workflows/check-dist.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+  workflow_dispatch:
```
Comment (alehechka) [inline .github/workflows/check-dist.yaml:13]: does this key need a value?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,25 @@
...
+
+jobs:
+  publish:
+    name: Update Doc
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update Doc
+        uses: dmsi-io/gha-docs-scoop@main
+        with:
+          repo: ${{ github.event.repository.name }}
```
Comment (alehechka) [inline README.md:23]: can you make this the default value and change required to false?

File: action.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      uses: actions/checkout@v2
+      with:
+        repo: 'dmsi-io/docs-hugo'
+        token: ${{ inputs.token }}
+    - name: Git Config
+      run: |
+        git config user.email github-actions@github.com
+        git config user.name github-actions
+      shell: bash
+      working-directory: docs-hugo
```
Comment (alehechka) [inline action.yaml:25]: does this need to be moved to the `Update Docs` step below?

Comment (alehechka) [reply action.yaml:25]: nevermind, I see how this `dist` is setup in this repo.

File: action.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    - name: Update Docs
+      run: node ./dist/index.js ${{ inputs.repo }}
```
Comment (alehechka) [inline action.yaml:27]: you'll need to use this to access you script `${{ github.action_path }}/dist/index.js`

File: .github/workflows/check-dist.yaml (hunk 1 above)
Comment (hbiede) [reply .github/workflows/check-dist.yaml:13]: This file was stole directly from a GitHub shared action

**Review by alehechka — APPROVED** [review]

**Review by Adam-Schlichtmann — APPROVED** [review]

**Pull request merged**